README.md

```diff
@@ -248,3 +248,16 @@ the available simulations are defined in `.js` files in `sims`.
 
 Add `-q` to quiet down the logging.
 Add `-o <output>` to output the simulation data to `<output>`.
+
+## UI
+
+A web interface can be used to visualize the output of a simulation.
+
+### Requirements
+
+The UI presently expects the output of the simulation to be stored in a file named `datastore.json` in the root
+repository.
+
+### Development
+
+Run `yarn start`.
```

Comment:
I think this is now for more than development!

To visualize the output of a simulation:

in one terminal:

```
yarn sim $SIMULATION -o datastore.json
```

Then, in a new terminal, run
```
yarn start
```
and navigate to ...?.  The page will automatically reload if `datastore.json` is modified, such as if you run a new simulation.